compiler/rustc_next_trait_solver/src/canonicalizer.rs

```diff
@@ -467,3 +471,82 @@ impl<I: Interner> TypeFolder<I> for RegionsToStatic<I> {
         }
     }
 }
+
+///////////////////////////////////////////////////////////////////////////
```

Comment:
can you move that into a separate module, e.g. `rustc_next_rait_solver/src/resolve.rs` or sth